src/org/traccar/notification/NotificationFormatter.java

```diff
@@ -70,6 +70,23 @@ private NotificationFormatter() {
             + "Point: http://www.openstreetmap.org/?mlat=%3$f&mlon=%4$f#map=16/%3$f/%4$f%n"
             + "Time: %2$tc%n";
 
+    public static final String TITLE_TEMPLATE_TYPE_ALARM = "%1$s: alarm!";
+    public static final String MESSAGE_TEMPLATE_TYPE_ALARM = "Device: %1$s%n"
+            + "Alarm: %5$s%n"
+            + "Point: http://www.openstreetmap.org/?mlat=%3$f&mlon=%4$f#map=16/%3$f/%4$f%n"
```

Comment:
Maybe we should have a special page in our system to show map instead of using OSM link? It definitely shouldn't be part of this pull request. Just an idea for future improvement.